pkg/lib/sidecars/proxy.go

```diff
@@ -12,8 +12,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	podsToRestartLimit = 30
```

Comment:
Does that mean that if there are 30 deployments containing 100 replicas each, we could in theory restart 3000 pods at once? Maybe we could sort the pods, so the restart would reflect the deployment size a bit more.

Reply:
Let me dig a bit more on that. Kind of analysing deployment on top of selecting X pods seems to me a bit too much for this task. Will check if there could be easy solution.